CONTRIBUTING.md

```diff
@@ -2,88 +2,156 @@
 
 Thank you for your interest in contributing to the Metaplex Aura Network! This document outlines our development process and guidelines.
 
+## Getting Started
+
+### Prerequisites
+
+-   **Rust:** This project requires Rust **version 1.84**. We recommend using `rustup` to manage your Rust installation. You can install `rustup` by following the instructions at [https://rustup.rs/](https://rustup.rs/). After installing `rustup`, ensure you have the correct version:
+
+    ```bash
+    rustup toolchain install 1.84
+    rustup default 1.84
+    ```
+
+-   **Docker and Docker Compose:** Many components of Aura are run within Docker containers. Install Docker and Docker Compose from [https://docs.docker.com/get-docker/](https://docs.docker.com/get-docker/).
+-   **System Libraries:** You may need to install additional system libraries depending on your operating system. (We should add specific instructions for common OSes here). Specifically, you'll need `protobuf-compiler`:
+    ```bash
+    sudo apt-get update && sudo apt-get install -y protobuf-compiler # Example for Debian/Ubuntu
+    ```
+-   **Git:** You'll need Git to clone the repository and manage your changes.
+
+### Setup
+
+1.  **Clone the Repository:**
+
+    ```bash
+    git clone <repository_url>
+    cd <repository_name>
+    ```
+
+2.  **Build the Project:**
+
+    ```bash
+    cargo build
+    ```
+
+    This will download dependencies and build the project. The first build may take a while.
+
+3.  **Run Unit Tests:**
+
+    ```bash
+    cargo test
+    ```
+
+4.  **Run Integration Tests:**
+
+    ```bash
+    cargo test -F integration_tests
+    ```
+
+    Note: Integration tests may require specific environment variables or setup. See the `README.md` and `docker-compose.yaml` for details. The integration tests use a local PostgreSQL instance, as configured in the `rust.yml` CI workflow.
```

Comment:
CI and local are different due to permissions; local tests, esp. the ones with synchronization, should be bootstrapped first and foremost with `integration_tests/run_postgres.sh`.